script/utils/transferOwnership.sh

```diff
@@ -0,0 +1,142 @@
+#!/bin/bash
```

Comment:
rearrange the file

constant
* add color constants
* function
* main logic / calling functions

Reply:
updated